fern/api/definition/api.yml

```diff
@@ -0,0 +1 @@
+name: api
```

Comment:
small: lets call this like `defnition-edit-api` or something?

Reply:
yeah was waiting on naming couldn't think of anything

Reply:
nbd but it's more than `edit` since it'll probably be all of CRUD on APIs